Ps3DiscDumper/DiscKeyProviders/IrdProvider.cs

```diff
@@ -74,7 +74,7 @@ public async Task<HashSet<DiscKeyInfo>> EnumerateAsync(string discKeyCachePath,
                 Log.Info($"If you have matching IRD file, you can put it in '{discKeyCachePath}' and try dumping the disc again");
             }
 
-            Log.Info($"Found {result.Count} IRD files");
+            Log.Info($"Found {result.Count} IRD files in the cache");
```

Comment:
this is a total number of loaded IRD files, if you want to log the number of cached irds, you can add a new line at [line 53](#diff-6effd2d0e5154a7b1fd52ee13129be12L53)